Extension/i18n/csy/src/SSH/sshCommandRunner.i18n.json

```diff
@@ -6,7 +6,7 @@
 {
 	"ssh.canceled": "Příkaz SSH je zrušený",
 	"ssh.passphrase.input.box": "Zadejte heslo pro klíč SSH {0}",
-	"ssh.enter.password.for.user": "Zadejte heslo pro uživatele {0}",
+	"ssh.enter.password.for.user": "Zadejte heslo pro uživatele{0}",
```

Comment:
did we want to delete spaces before substitutions `{}` in this file?

Reply:
@bobbrow I already filed a bug on the spaces issues. I'm assuming that any remaining issues with spaces is "by design" for the language unless we get more info.